src/index.html

```diff
@@ -7,8 +7,68 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img class="logo" src="logo.png" alt="logo">
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item" >
+            <a class="nav__link is-active" href="#">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="hover"
+              href="#"
+            >
+              Laptops &amp; computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+            Gadgets
+            </a>
```

Comment:
Do not forget about 2 spaces
```suggestion
            <a class="nav__link" href="#">
              Gadgets
            </a>
```